isis/scripts/install_isis.sh

```diff
@@ -201,36 +213,40 @@ case "$(uname)" in
         ;;
 esac
 
-# Check if conda is installed but mamba is not
+# Check if conda is installed but mamba is not, or if forced to install mamba
 if command -v conda &> /dev/null && ! command -v mamba &> /dev/null; then
-    echo "WARNING: conda is installed but mamba is not."
-    echo "mamba is a faster alternative to conda and is required for ISIS installation."
-    echo "Otherwise, environment creation will take much longer than what is practical."
-    echo "Would you like to install mamba? [yes|no]"
-    ans="yes"
-    printf "[%s] >>> " "$ans"
-    
-    read -r ans
-    
-    # If no input, use default
-    if [ "$ans" = "" ]; then
+    if [ "$FORCE_MAMBA_INSTALL" = "YES" ]; then
+        echo "Forcing mamba installation."
+    else
+        echo "WARNING: conda is installed but mamba is not."
+        echo "mamba is a faster alternative to conda and is required for ISIS installation."
+        echo "Otherwise, environment creation will take much longer than what is practical."
+        echo "Would you like to install mamba? [yes|no]"
```

Comment:
This seems counter to what will actually happen if you say you don't want to install mamba. The script will just exit rather than even trying to install ISIS using conda. Either the script should continue without mamba or this message should be change saying the script will exit if you choose to not install mamba

Reply:
hmmm, I thought I changed this, maybe a weird merging thing

Reply:
The comment should likely read something more like
```
WARNING: conda is installed but mamba is not.
This script requires mamba be installed, if you choose to not install it
this script will terminate
```

I might be missing something but the script won't let you create an ISIS environment without mamba installed

Reply:
updated